packages/runtime/container-runtime/src/summary/summaryDelayLoadedModule/summaryResultBuilder.ts

```diff
@@ -0,0 +1,154 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert, Deferred } from "@fluidframework/core-utils/internal";
+
+import {
+	SubmitSummaryFailureData,
+	SubmitSummaryResult,
+	SummarizeResultPart,
+	type IRetriableFailureError,
+} from "../summarizerTypes.js";
+
+import type {
+	ISummaryAckMessage,
+	ISummaryNackMessage,
+	ISummaryOpMessage,
+} from "./summaryCollection.js";
+
+/**
+ * @legacy
+ * @alpha
+ */
+export interface IBroadcastSummaryResult {
```

Comment:
These types are not specific to the `SummarizeResultBuilder` so having them in that file is confusing. I would suggest moving them to a file called `summaryResultTypes` or something like that.

`SummarizeResultBuilder` can be in this file as a single utility which is fine.

Reply:
done